docs/source/conf.py

```diff
@@ -147,96 +84,110 @@
-htmlhelp_basename = "condadoc"
 
+# -- sphinxcontrib.plantuml ------------------------------------------------
 
-# -- Options for LaTeX output ---------------------------------------------
+plantuml_output_format = "svg_img"
+plantuml_jarfile_path = Path(__file__).parent.parent / "_build" / "plantuml.jar"
+plantuml = f"java -Djava.awt.headless=true -jar {plantuml_jarfile_path}"
 
-latex_elements = {
-    # The paper size ('letterpaper' or 'a4paper').
-    #
-    # 'papersize': 'letterpaper',
-    # The font size ('10pt', '11pt' or '12pt').
-    #
-    # 'pointsize': '10pt',
-    # Additional stuff for the LaTeX preamble.
-    #
-    # 'preamble': '',
-    # Latex figure (float) alignment
-    #
-    # 'figure_align': 'htbp',
-}
 
-# Grouping the document tree into LaTeX files. List of tuples
-# (source start file, target name, title,
-#  author, documentclass [howto, manual, or own class]).
-latex_documents = [
-    (master_doc, "conda.tex", "conda Documentation", "Anaconda, Inc", "manual"),
-]
+# -- For sphinx.ext.intersphinx --------------------------------------------
 
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3", None),
+    "pluggy": ("https://pluggy.readthedocs.io/en/stable/", None),
+}
 
-# -- Options for manual page output ---------------------------------------
 
-# One entry per manual page. List of tuples
-# (source start file, name, description, authors, manual section).
-man_pages = [(master_doc, "conda", "conda Documentation", [author], 1)]
+# -- For sphinx_sitemap ----------------------------------------------------
 
+sitemap_locales = [None]
+sitemap_url_scheme = "{lang}stable/{link}"
 
-# -- Options for Texinfo output -------------------------------------------
 
-# Grouping the document tree into Texinfo files. List of tuples
-# (source start file, target name, title, author,
-#  dir menu entry, description, category)
-texinfo_documents = [
-    (
-        master_doc,
-        "conda",
-        "conda Documentation",
-        author,
-        "conda",
-        "One line description of project.",
-        "Miscellaneous",
-    ),
-]
+# -- For myst_parser -------------------------------------------------------
 
-plantuml_output_format = "svg_img"
-plantuml_jarfile_path = os.path.abspath(
-    os.path.join(os.path.dirname(__file__), "..", "_build", "plantuml.jar")
-)
-plantuml = f"java -Djava.awt.headless=true -jar {plantuml_jarfile_path}"
-
-add_module_names = False
+myst_heading_anchors = 3
+myst_enable_extensions = [
+    "amsmath",
+    "colon_fence",
+    "deflist",
+    "dollarmath",
+    "html_admonition",
+    "html_image",
+    "linkify",
+    "replacements",
+    "smartquotes",
+    "substitution",
+    "tasklist",
+]
 
 
-intersphinx_mapping = {
-    "python": ("https://docs.python.org/3", None),
-    "pluggy": ("https://pluggy.readthedocs.io/en/stable/", None),
+# -- For autoapi.extension -------------------------------------------------
+
+autoapi_dirs = ["../../conda", "../../conda_env"]
```

Comment:
Are these relative to `conf.py` or the working directory?

Reply:
https://sphinx-autoapi.readthedocs.io/en/latest/reference/config.html#confval-autoapi_dirs:

> Relative paths should be relative to the source directory of your documentation.